src/components/Result/index.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import "./style.css";
+
+export default function renderResult({ result }) {
+  const renderList = () => {
+    // console.log(Object.values(result.repos));
+    const repoObjectArray = [];
+    for (const property in result.repos) {
+      const entry = Object.values(result.repos)[property];
+      repoObjectArray.push({
+        name: entry.name,
+        forks: entry.forks,
+        url: entry.html_url
+      });
+    }
+
+    return (
+      <div id="table">
+        <table>
+          <thead>
+            <tr>
+              <th>Repository Name</th>
+              <th>Number of Forks</th>
+            </tr>
+          </thead>
+          <tbody>
+            {repoObjectArray.map((repo, id) => (
+              <tr key={id}>
+                <td>
+                  <a href={repo.url}>{repo.name}</a>
```

Comment:
When we clicked this we had the console open and it logged a bunch of cors warnings. Idk if theres anything you can do about that tho
![image](https://user-images.githubusercontent.com/91017452/157636853-89cc3f6b-96e9-4752-b729-1187daf273c5.png)